src/components/settings/tests/SettingsTabs.test.jsx

```diff
@@ -32,7 +32,7 @@ const settingsTabsWithRouter = () => (
   </MemoryRouter>
 );
 
-describe('<SettingsTabs />', () => {
+describe('<SettingsPage />', () => {
```

Comment:
nit: Why do we describe these tests as `<SettingsPage />` when we are testing `<SettingsTabs />` on this page? Should we just name `SettingsTab` to `SettingsPage`?

Reply:
D'oh, good catch. This wasn't meant to be changed! Changing it back to `SettingsTabs`. We already have a `SettingsPage` component (index.jsx).